app/views/content_items/travel_advice.html.erb

```diff
@@ -26,23 +26,9 @@
       </div>
       <div class="travel-advice-notice__content">
         <% if @content_item.ireland? %>
-          <%= render 'content_items/ireland_covid_travel_guidance' %>
+          <%= render_govspeak render('content_items/travel_advice/ireland_covid_travel_guidance.md') %>
```

Comment:
A thing to note is that you're suffixing this file with md to imply it's a markdown file but then the function is treating the content as govspeak.

It might be overkill to have govspeak in the mix and better to just have a render_markdown function. That'd save having to bring over all the other dependencies that come with govspeak - similar to how we do this on https://github.com/alphagov/govuk_publishing_components/blob/5d4ed0788b5dd2a9865b45eb7d062eb7f59dd73a/app/models/govuk_publishing_components/component_doc.rb#L86-L88

Reply:
Ahh that's a great suggestion, thanks Kevin!